workflows/scATAC_seq/Snakefile

```diff
@@ -0,0 +1,37 @@
+import re
+
+# complement the configuration
+configfile: "config.yaml"
+
+# do onstart/onexit things
+sample_schemas = ['sample', 'assembly']
+config_schemas = ['general', 'download', 'alignment_general', 'alignment_specific']
+include: "../../rules/configuration.smk"
+
+
+# load all the relevant rules
+include: f"{config['rule_dir']}/alignment.smk"
+include: f"{config['rule_dir']}/get_fastq.smk"
+include: f"{config['rule_dir']}/get_genome.smk"
+include: f"{config['rule_dir']}/merge_replicates.smk"
+include: f"{config['rule_dir']}/qc.smk"
+include: f"{config['rule_dir']}/scATAC.smk"
+include: f"{config['rule_dir']}/trimming.smk"
+
+# set the quality_control functions
+quality_control = [get_trimming_qc, get_alignment_qc]
+
+
+rule snapATAC_all:
+    """
+    """
+    input:
+        expand((["{qc_dir}/multiqc_{assemblies}.html"] if config["create_qc_report"] else []) +
+                [f"{{result_dir}}/snap/{trep}-{treps.loc[trep, 'assembly']}.binned.snap" for trep in treps.index],
+                 **{**config,
+                    **{'assemblies': set(samples['assembly'])}})
```

Comment:
No trackhub wanted?

Reply:
Hmmm, not sure. Perhaps similar to RNAseq that by default it is off but you can turn it on? Because for some technical replica's where it's the same cell type it could be a quick QC check to look if the track seems nice. However when cell types are mixed on plates it would require first R snapatac tools (clustering and viz of the cluster signals) before they make sense.

Reply:
Both are fine by me. It's also something we can always add later.